spec/src/main/asciidoc/entities.asciidoc

```diff
@@ -56,7 +56,7 @@ Implementations may define additional custom scalars beyond those listed above.
 
 ==== Numbers
 All number scalars (Int, Double, Float, Short, Long, Byte, BigInteger and BigDecimal) can be formatted
-using the JSON-B annotation `@JsonbNumberFormat`.
+using the `@NumberFormat` or alternativly the JSON-B annotation `@JsonbNumberFormat` (except that `@JsonbNumberFormat` is not valid on `TYPE_USE`)
```

Comment:
Should we mention a priority in the event that a field/method/parameter/etc. is annotated with both `@NumberFormat(XYZ)` and `@JsonbNumberFormat(ABC)`?  In that case, I think that the `@NumberFormat` from MP GraphQL should win, but we might need to specify it here.  And maybe add a TCK test for it too?

Reply:
Good point. Done